docs/content.go

```diff
@@ -715,10 +715,39 @@ Enterprise Performance Computing (EPC)`
 	InspectUse   string = `inspect [inspect options...] <image path>`
 	InspectShort string = `Show metadata for an image`
 	InspectLong  string = `
-  Inspect will show you labels, environment variables, and scripts associated 
-  with the image determined by the flags you pass.`
+  Inspect will show you labels, environment variables, apps and scripts associated 
+  with the image determined by the flags you pass.
+  `
```

Comment:
I would possibly tell the user here what the default is. E.g. (and I'm not sure what the default is so fix this appropriately),

> By default, if you don't specify flags or format you will see the container's labels in json format.